### PR TITLE
fix/duplicate-name-fix

### PR DESCRIPTION
**Pull request by kdanilenka32**

### CODE REVIEW

**Review by wildkiomi — APPROVED** [review]

File: src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/ReportTable/ReportTable.const.ts [hunk 1]
```diff
@@ -22,3 +22,50 @@ export const getHeadCells = (): HeadCell[] => {
     },
   ];
 };
+
+export const rows = [
```
Comment (nmialik) [inline src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/ReportTable/ReportTable.const.ts:26]: I suppose it should be in another PR?)

Comment (nmialik) [inline src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/ReportTable/ReportTable.const.ts:26]: Should it be in this PR?

**Review by IhorNesterenko99 — APPROVED** [review]
